lektor_s3.py

```diff
@@ -189,4 +192,23 @@ def publish(self, target_url, credentials=None, **extra):
             yield 'deleting %s' % f
         self.delete_batch(diff['delete'])
 
-
+        # should we invalidate cloudfront?
+        # only invalidate added and updated files; leave deleted as they are
```

Comment:
Why do deleted items get to persist? Is this a cloudfront thing? I have no real experience with cloudfront.